Add Data Transfer user documentation (#52)
EGI-Foundation/documentation: content/en/users/datatransfer/_index.md

## Overview 

The EGI Data Transfer Service is based on the FTS3 service, developed at CERN. It allows you to move any type of data files asynchronously from one storage to another. The service includes dedicated interfaces to display statistics of on-going transfers and manage storage resources parameters.

The EGI  Data Transfer is ideal to move large amounts of files or very large files as the service has mechanisms to verify checksums and ensure automatic retry in case of failures.

## Features

Simplicity

: Easy user interfaces for submitting transfers:  CLI, Python Bindings, WebFTS and Web Monitoring.  

Reliability

: Checksums and retries are provided per transfer

Flexibility

: Multi-protocol support (Webdav/https, GridFTP, xrootd, SRM, S3, GCloud). 

Intelligence

: Parallel transfers optimization to get the most from network without burning the storages. Priorities/Activities support for transfers classification. 

## Components 

FTS3 Server 

: The service is responsible of the asynchronous execution of the file transfer, checksumming and retries in case of errors

FTS3 REST  

: The RESTFul server which is contacted by clients via REST APIs, CLI and Python bindings

> 

Reviewer: ```suggestion
```

---
title: "Data Transfer"
linkTitle: "Data Transfer"
type: docs
description: "Documentation related to EGI Data Transfer"
weight: 30
---

## Overview 

The EGI Data Transfer Service is based on the FTS3 service, developed at CERN. It allows you to move any type of data files asynchronously from one storage to another. The service includes dedicated interfaces to display statistics of on-going transfers and manage storage resources parameters.

> The EGI  Data Transfer is ideal to move large amounts of files or very large files as the service has mechanisms to ensure automatic retry in case of failures and checksumming.

Reviewer: ```suggestion
The EGI  Data Transfer is ideal to move large amounts of files or very large files as the service has mechanisms to verify checksums and ensure automatic retry in case of failures.
```

> 

Reviewer: ```suggestion
```

The service includes dedicated interfaces to display statistics of on-going transfers
and manage storage resources parameters.

The EGI  Data Transfer is ideal to move large amounts of files or very large files as
the service has mechanisms to verify checksums and ensure automatic retry in case of 
failures.

## Features

Simplicity

: Easy user interfaces for submitting transfers:  CLI, Python Bindings, WebFTS and Web 
Monitoring.  

Reliability

: Checksums and retries are provided per transfer

Flexibility

: Multi-protocol support (Webdav/https, GridFTP, xrootd, SRM, S3, GCloud). 

Intelligence

: Parallel transfers optimization to get the most from network without burning the storages.
 Priorities/Activities support for transfers classification. 

## Components 

FTS3 Server 

: The service is responsible of the asynchronous execution of the file transfer, checksumming 
and retries in case of errors

FTS3 REST  

: The RESTFul server which is contacted by clients via REST APIs, CLI and Python bindings

FTS3 Monitoring  

: A Web interface to  monitor transfers activity and server parameters

>  

Reviewer: ```suggestion

```
